base_synchro/wizard/base_synchro.py

```diff
@@ -28,7 +28,9 @@
 from openerp import tools
 import threading
 import unicodedata
-from openerp.osv import fields, osv
+from openerp import models, fields, api, _
+from openerp.tools.misc import frozendict
```

Comment:
Improve import statements and remove unused imports.
ex: from openerp.tools import frozendict